docs/index.rst

```diff
@@ -0,0 +1,81 @@
+.. Images
+.. |asset| image:: images/asset.jpg
+
+
+Fledge Asset Filter
```

Comment:
I see some of the filters are having prefix "Fledge" and some are not (e.g change, delta, metadata etc..).
I believe we should remove this prefix to make it consistent across all plugins.

Reply:
Sure